src/transformStateWithClones.js

```diff
@@ -65,7 +65,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const cloneState = { ...state };
+  const arrWithCloneState = [];
+
+  transforms.forEach((element) => {
```

Comment:
```suggestion
  transforms.forEach((transform) => {
```
берешь одну трансформацию с массива transforms